packages/client/src/components/formConfig/FormConfigCanvas.tsx

```diff
@@ -0,0 +1,30 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * OpenCRVS is also distributed under the terms of the Civil Registration
+ * & Healthcare Disclaimer located at http://opencrvs.org/license.
+ *
+ * Copyright (C) The OpenCRVS Authors. OpenCRVS and the OpenCRVS
+ * graphic logo are (registered/a) trademark(s) of Plan International.
+ */
+
+import React from 'react'
+import styled from '@client/styledComponents'
+
+const Container = styled.div`
+  margin-left: 330px;
+  margin-right: 438px;
+  margin-top: 74px;
```

Comment:
I think you need to refactor these styles a little. If we look at the container for the two sidebars and this Canvas, we see that the container does not contain the three elements but stays collapsed below the header (partially overlapping it).

<img width="1597" alt="image" src="https://user-images.githubusercontent.com/1206987/161753798-bb94e562-9cda-4ec6-8010-a64af2195bc9.png">

It's essential that elements flow correctly (appear inside of each other in DevTools) because doing that gives you 80% of responsivity out of the box. You dodge the majority of layout bugs, especially when thinking in terms of supporting many different browsers.

I think the correct way of building the layout here would be to remove the `position: fixed` attributes altogether and do a flex layout of

```
[header]
[ [left sidebar] [canvas] [right sidebar] ]
```

Example:

![Kapture 2022-04-05 at 15 37 32](https://user-images.githubusercontent.com/1206987/161755261-6a38e457-0b7b-4209-bb1e-d9b555760708.gif)

Reply:
@rikukissa This would have been the ideal case but only the canvas should be scroll able so we need to have fixed position for the container and sidebars

Reply:
It's best to make the canvas `overflow: auto` in that case. When its content exceeds the height of the window, you'll get a scroll bar and all the other parts of the view remain in their positions

Reply:
Yeah that should work. Only that the `EventTopBar` which is the header, itself has `position: fixed` and other things might break if we change that so I guess for now keeping the header as it is and wrapping the sidebars and canvas in a flex seems feasible. What do you think?

Reply:
Yeah 👍 Definitely reasonable.